docs/src/domains/cdn/fastly.md

```diff
@@ -1,55 +1,62 @@
 ---
-title: "Fastly"
+title: "Configure your Fastly CDN"
+sidebarTitle: "Fastly"
 weight: 1
-description: In some cases you may want to opt to use a CDN such as Fastly rather than the Platform.sh router's cache. Using a CDN can offer a better time-to-first-byte for cached content across a wider geographic region at the cost of the CDN service.
+description: Learn how to configure your Fastly CDN.
+aliases:
+  - "/golive/steps/fastly.html"
 ---
 
-{{% description %}}
+You can [use a CDN](./_index.md) to deliver your site's content to users more quickly.
 
-A Fastly CDN is included for Dedicated projects.
-Platform.sh doesn't offer an integrated CDN on self-service Grid projects at this time,
-but it is a common choice for customers to self-configure.
+By default, a Fastly CDN is included in Dedicated projects.
```

Comment:
:lady_beetle: If the assumptions I made in the open question from the last review are true, then this entire page doesn't apply to Dedicated projects. But I don't think that is clear here. I think there should be something clearer about what Dedicated users need to do, if anything. I think it's only the Anycast support ticket, if applicable.

Reply:
You are right in that most of the page isn't for Dedicated projects. However, there's information about CNAME records being provided by PSH that seems important to share with Dedicated users as it's specific to them. I moved it to the beginning of the page to clearly differentiate Dedicated from Grid (=> the rest of the page) but I'm not entirely sure it's the best place to put it.